I am a 6th year PhD candidate in the Center for Computational Biology at the University of California, Berkeley. I am advised by [Dr Liana Lareau](https://lareaulab.org/) and [Dr Nir Yosef](https://yoseflab.github.io). My research focuses in analysing alternative splicing in single cell RNA-sequencing (scRNA-seq) data, and in the regulation of splicing of ultraconserved poison exons.

I recieved my BSc in Biology from Universidad Nacional Autonoma de Mexico in 2015.

### Alternative splicing in single cells

Observations of alternative splicing with scRNA-seq led to an unexpected observation: in a homogeneous population of single cells, most cells consistently show either one isoform or another of the same gene, but they rarely produce both. In other words, the apparent distribution of the Percent Spliced-In (&Psi) of isoforms is bimodal and mostly binary. In [recent work](https://elifesciences.org/articles/54603), I showed that the distortion of the observed $Psi by low capture efficiency and amplification biases are the main source of this apparent bimodality in full-transcript scRNA-seq data. The challenge for Computational Biologists ahead is to account for these biases in the analysis of informative alternative splicing at the single cell level.

### Regulation of poison exon splicing

[Ultraconserved poison exons](https://www.nature.com/articles/nature05676) are a fascinating yet largely unknown mechanism of post-transcriptional regulation in the Serine/Argenine-Rich (SR) family of splicing factors. I am currently working on analysing the binding patterns of RNA Binding Proteins (RBPs) in these exons in order to reveal potential regulatory mechanisms.
